### ISSUE TITLE
Help on library functions is not available if the function name has a question mark (Github web page)

### ISSUE THREAD

**Issue by progamateur** [arturo-lang/arturo#317]
The page https://github.com/arturo-lang/arturo/wiki/Library is concerned.

The page https://arturo-lang.io/documentation/library/, which gives the same possibilities of help does not have this problem.

To check it, in the paragraph "Alphabetical Index" click on "and?" for example and see the result for both pages.

Browsers tested and concerned by this problem:
- Windows 10 : Opera, Brave, Edge
- Android: Opera

**Comment by drkameleon** [comment]
Ooops. I had totally missed this one. I don't know when I introduced this error, I hope it wasn't there from the beginning.

The thing with the wiki documentation right now is... that I've been contemplating to deprecate it. I mean, since the documentation at the "official" website is - finally - quite complete, it doesn't make a lot of sense to have the same things twice, in two different places, and with a slightly similar (in this case, worse) layout, does it?

Basically, what I was thinking, is keep a folder at the main repo with the wiki files, and just have a message at the present wiki pointing to the newest version of the documentation, that is: at the website...

What do you think?

**Comment by progamateur** [comment]
I agree 100%

**Comment by drkameleon** [comment]
> I agree 100%

Perfect.

Let's see how it goes. I've been planning to do exactly that for some time, but I guess I have to somehow find the time to do aaaall of that... lol
